sql/sql_parse.cc

```diff
@@ -2572,7 +2572,7 @@ static bool lock_tables_for_backup(THD *thd)
   @return FALSE in case of success, TRUE in case of error.
 */
 
-static bool lock_binlog_for_backup(THD *thd)
+bool lock_binlog_for_backup(THD *thd)
```

Comment:
These changes, being up in the server code, will need a small blueprints to attach to for 5.6 and 5.7